nncf/torch/dynamic_graph/wrappers.py

```diff
@@ -205,18 +205,25 @@ def _collect_module_attrs_and_ignored_algorithms(
     layer_attrs = None
     ignored_algos = []
     from nncf.torch.graph.operator_metatypes import OP_NAMES_WITH_WEIGHTS
+    from nncf.torch.nncf_network import NNCFNetwork
 
-    if op_name in OP_NAMES_WITH_WEIGHTS:
-        curr_module = ctx.get_current_module()
-        if curr_module is None:
-            raise nncf.ValidationError(
-                f"Operation {op_name} requires module attributes, but it was executed outside any module"
-            )
-        layer_attrs = get_layer_attributes_from_module(curr_module, op_name)
-        if isinstance(curr_module, _NNCFModuleMixin):
-            ignored_algos = deepcopy(curr_module.ignored_algorithms)
-    elif op_name in OP_NAMES_REQUIRING_ATTRS_FROM_ARGS_KWARGS:
+    if (
```

Comment:
Do you have a summary table of what metatypes are supported layer attributes collection if trace_parameters == True and False?

Reply:
No, but it's align for both options.

Reply:
Could you merge it If it's align?

Reply:
done